governance/parameter-docs/param-liquidation-penalty.md

```diff
@@ -44,7 +44,7 @@ Decreasing the Liquidation Penalty should result in less system revenue during l
 
 ## Considerations
 
-Adjusting the Liquidation Penalty parameter should always be done with great care. As this parameter directly affects the risk of vault holders and the system, changes in either direction could be met with negative consequences. 
+Adjusting the Liquidation Penalty parameter should always be done with great care. As this parameter directly affects the risk of vault holders and the system, changes in either direction could be met with negative consequences. The Liquidation Penalty should always be higher than liquidation rewards so as to discourage incentive farming.
```

Comment:
New section could maybe be clearer? What is incentive farming? I would like to keep it to one sentence though.

Reply:
Prior sentence seems like a bit of a space filler too, maybe we don't need that. Not sure.